src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+
+const ChatLog = (props) => {
+  const messageComponents = props.entries.map(message => {
+    return (
+      <ChatEntry className="chat-log"
+        key={message.id}
+        id={message.id}
+        sender={message.sender}
+        body={message.body}
+        timeStamp={message.timeStamp}
+        liked={message.liked}
+        onLikeToggle={props.onLikeToggle}
+      ></ChatEntry>
+    );
+  });
+
+  return (
+    <>
+      {messageComponents}
+    </>
+  );
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
+      sender: PropTypes.string.isRequired,
+      body: PropTypes.string.isRequired,
+      timeStamp: PropTypes.string.isRequired,
+      id: PropTypes.number,
+      liked: PropTypes.bool
+    })
+  ).isRequired,
+  onLikeToggle: PropTypes.func
+};
```

Comment:
Nice work including prop types for this component! Just a quick reminder: in React v19, `PropTypes` are deprecated. Moving forward, it's best to use `TypeScript` for type checking, as it's the recommended and more robust solution for ensuring type safety across your application.